macosx/CKAN

```diff
@@ -22,12 +22,25 @@ else
     # Mono found, so we can run CKAN
     # The exe is called ckan.exe
     ASSEMBLY=ckan.exe
+
+    MONO_ARGS="--arch=32"
+    for ARG in "$@"
+    do
+        # If we want to run the GUI, set mono to 32bit mode. If not, go 64bit.
+        if [[ ! $ARG =~ ^- && $ARG != gui ]]
+        then
+            MONO_ARGS="--arch=64"
```

Comment:
What do you think about clearing the variable instead, in case there's a user out there with a 32-bit only Mac from 2006?

Reply:
I didn't want to do that because of this:
> the execution switches to a binary with the selected bitness suffix installed side by side (for example, '/bin/mono --arch=64' will switch to '/bin/mono64' if '/bin/mono' is a 32-bit build).

So to _force_ the 64 bitness, no matter how it's installed.

The help text also says:
> Selects the bitness of the Mono binary used, **if available**

That sounds like if 64-bit mono isn't available, the command will be ignored.